app-api/src/main/java/org/openmrs/eip/app/management/entity/ReceiverSyncHistory.java

```diff
@@ -0,0 +1,145 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
```

Comment:
Why do you need a new table?

Reply:
Wyclif, 
The purpose of this ticket is to have all events sent to the central server by a remote site
for historical purposes in order to resend these events if necessary.

We decide to create new tables because the table created by Luis Pinto, SyncMessage for sender side, it deletes a row of that events when it receives the response from the server, while in the UI implementations i need to display all event : NEW , SENT and RECEIVED events.